dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsWrapper.java

```diff
@@ -124,24 +126,46 @@ public String getConfiguredLocaleName(String localeCode) {
         return configuredLocales.get(localeCode);
     }
 
+    public Map<String, String> getConfiguredAboutUrls() {
+        if (configuredAboutUrls == null) {
+            initConfiguredAboutUrls();
+        }
+        return configuredAboutUrls;
+    }
+
     // -------------------- PRIVATE --------------------
 
     private void initLocaleSettings() {
+        configuredLocales = initMapFromJson(SettingsServiceBean.Key.Languages, "locale", "title");
+    }
 
-        configuredLocales = new LinkedHashMap<>();
+    private void initConfiguredAboutUrls() {
+        configuredAboutUrls = initMapFromJson(SettingsServiceBean.Key.NavbarAboutUrl, "url", "title");
+    }
 
+    private Map<String, String> initMapFromJson(SettingsServiceBean.Key settingKey, String mapKey, String mapValue) {
```

Comment:
Since creating a Json object is an implementation detail in this method, I would rather call it something like `initMapForSettingKey` or `getSettingMapForKey` or something along those lines.

Reply:
As the format of the settings file is `*.properties` file, it has it's limitation. JSON format was used as some properties values as a kind of extension to that format. Being explicit about putting JSON inside `*.properties` file doesn't sound like a bad idea to me, as it's rather unexpected.
Also, this method relies on the setting to be in JSON format. This method can only read JSON, so if you call it, be sure that this setting is in JSON format, not any `String` that can be in your `*.properties` file. It can read any setting, as long as it's in JSON format.
Are you sure you'd want to hide that info from the caller? Especially that it's a private method anyways, and the caller needs to already be aware of the structure of the settings file?